kinetic/distribution.yaml

```diff
@@ -8076,6 +8076,22 @@ repositories:
       url: https://github.com/ros-planning/random_numbers.git
       version: master
     status: maintained
+  raspi_temperature:
+    doc:
+      type: git
+      url: https://github.com/bberrevoets/raspi_temperature.git
+      version: 0.1.1
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/bberrevoets/raspi_temperature-release.git
+      version: 0.1.1-0
+    source:
+      test_pull_requests: true
```

Comment:
Have you followed the [instructions](http://wiki.ros.org/buildfarm/Pull%20request%20testing) to provide access to this repo or the org unit to the @ros-pull-request-builder user? If not, either remove this line or provide the necessary access as described in the wiki page.

Reply:
I have corrected this error for the access to the repo I think, but how can I correct the valid branch name?

Reply:
You can edit the file in your fork (in the branch `bloom-raspi_temperature-0` as shown at the top of this PR) and commit the change. That commit will then show up in this PR.